applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_explicit_dynamic_solver.py

```diff
@@ -96,18 +96,25 @@ def _CreateScheme(self):
         scheme_type = self.settings["scheme_type"].GetString()
 
         # Setting the Rayleigh damping parameters
+        rayleigh_alpha = self.settings["rayleigh_alpha"].GetDouble()
+        rayleigh_beta = self.settings["rayleigh_beta"].GetDouble()
+        for prop in self.main_model_part.Properties:
+            prop.SetValue(StructuralMechanicsApplication.RAYLEIGH_ALPHA], rayleigh_alpha)
+            prop.SetValue(StructuralMechanicsApplication.RAYLEIGH_BETA], rayleigh_beta)
+
+        # Setting the Rayleigh damping parameters (legacy on ProcessInfo)
```

Comment:
This is not legacy!

Please check the implicit solver, there is an established workflow

Reply:
We should make it legacy for inconsistent

Reply:
And as I said, explicit cannot be totally consistent with implicit because otherwise computational cost will be huge

Reply:
I took a look now. The idea with the damping is that one can specify it globally (via `solver_settings` aka `ProcessInfo`), or locally (via the `Properties`). The locally defined ones take precedence

What one does with that is up to the user. I dont like the idea of excluding some stuff upfront, just because it might be expensive.

Keep in mind that Rayleigh damping is after all an estimation of the real structural damping.